source/MRViewer/ImGuiMenu.h

```diff
@@ -167,6 +167,8 @@ class MRVIEWER_CLASS ImGuiMenu : public MR::ViewerPlugin,
 
   MRVIEWER_API void draw_helpers();
 
+  MRVIEWER_API virtual void drawModalMessage();
```

Comment:
i think this function should be protected

Reply:
fixed